src/mumble/TalkingUI.cpp

```diff
@@ -569,6 +571,41 @@ void TalkingUI::mousePressEvent(QMouseEvent *event) {
 	updateUI();
 }
 
+void TalkingUI::wheelEvent(QWheelEvent *event) {
```

Comment:
This brings me to my next consideration: Why are we only allowing this in the Talking UI? It feels like the main user list could also profit from this. Maybe I am missing something you already considered?